<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>1024</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <kerning>false</kerning>
   </font>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLabel" name="ground">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>930</y>
     <width>2048</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <underline>false</underline>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>11111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::TextSelectableByMouse</set>
   </property>
   <property name="buddy">
    <cstring></cstring>
   </property>
  </widget>
  <widget class="QLabel" name="ground2">
   <property name="geometry">
    <rect>
     <x>2048</x>
     <y>930</y>
     <width>2048</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <underline>false</underline>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="text">
    <string>11111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111</string>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::NoTextInteraction</set>
   </property>
  </widget>
  <widget class="QLabel" name="ea4_2">
   <property name="geometry">
    <rect>
     <x>2050</x>
     <y>680</y>
     <width>2048</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <underline>false</underline>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
   </property>
   <property name="buddy">
    <cstring></cstring>
   </property>
  </widget>
  <widget class="QGroupBox" name="lol">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>943</y>
     <width>2048</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="ea6">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>2048</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <underline>false</underline>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>@###############@@@@@@@@@@@@@@@@@@@@@ ########### #@############## @#@@##@#@#@##@#@@#@@#@@@@@@@@@@@@@@@@##############@#@#@@@@@@@@@@@@@@@@@@@#@#@###########@#@#@############@#@#@#@############@@@@#@#@@@@@@@@@@@@#@#@@@@@@#@#@</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
    </property>
    <property name="buddy">
     <cstring></cstring>
    </property>
   </widget>
   <widget class="QLabel" name="ea2_3">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>10</y>
      <width>2048</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <underline>false</underline>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>@@@@########@#@#@@@@@@@@@@@@@@@@@@@@@ ########### #@############## @#@@##@#@#@##@#@@#@@#@@@@@@@@@@@@@@@@##############@#@#@@@@@@@@@@@@@@@@@@@#@#@###########@#@#@############@#@#@#@############@@@@#@#@@@@@@@@@@@@#@#@@@@@@#@#@</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
    </property>
    <property name="buddy">
     <cstring></cstring>
    </property>
   </widget>
   <widget class="QLabel" name="ea2">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>20</y>
      <width>2048</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <underline>false</underline>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>@@@@@@@@@@@@@@@@@@@@@ ########### #@############## @#@@##@#@#@##@#@@#@@#@@@@@@@@@@@@@@@@##############@#@#@@@@@@@@@@@@@@@@@@@#@#@###########@#@#@############@#@#@#@############@@@@#@#@@@@@@@@@@@@#@#@@@@@@#@#@</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
    </property>
    <property name="buddy">
     <cstring></cstring>
    </property>
   </widget>
   <widget class="QLabel" name="ea">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>2048</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <underline>false</underline>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>@@@@@@@@@@@@@@@@@@@@@ ########### #@############## @#@@##@#@#@##@#@@#@@#@@@@@@@@@@@@@@@@##############@#@#@@@@@@@@@@@@@@@@@@@#@#@###########@#@#@############@#@#@#@############@@@@#@#@@@@@@@@@@@@#@#@@@@@@#@#@</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
    </property>
    <property name="buddy">
     <cstring></cstring>
    </property>
   </widget>
   <widget class="QLabel" name="ea4">
    <property name="geometry">
     <rect>
      <x>-30</x>
      <y>40</y>
      <width>2048</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <underline>false</underline>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>@#@#@@@@@@@@@@@@@@@@@@@@@@##############@@@@@@@@@@@@@@@@@@@@@ ########### #@############## @#@@##@#@#@##@#@@#@@#@@@@@@@@@@@@@@@@##############@#@#@@@@@@@@@@@@@@@@@@@#@#@###########@#@#@############@#@#@#@############@@@@#@#@@@@@@@@@@@@#@#@@@@@@#@#@</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
    </property>
    <property name="buddy">
     <cstring></cstring>
    </property>
   </widget>
   <widget class="QLabel" name="ea5">
    <property name="geometry">
     <rect>
      <x>-50</x>
      <y>50</y>
      <width>2048</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <underline>false</underline>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>@#@@@@@@@@@@@@@@############@@@@@@@@@@@@##########@@@@@@@@@@@@@@@@@@@@@ ########### #@############## @#@@##@#@#@##@#@@#@@#@@@@@@@@@@@@@@@@##############@#@#@@@@@@@@@@@@@@@@@@@#@#@###########@#@#@############@#@#@#@############@@@@#@#@@@@@@@@@@@@#@#@@@@@@#@#@</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
    </property>
    <property name="buddy">
     <cstring></cstring>
    </property>
   </widget>
   <widget class="QLabel" name="ea5q">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>2048</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <underline>false</underline>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>@@@@@########@@@@@@#######@@@@@#@@@@@@@@@@@@@@@@@@@@@ ########### #@############## @#@@##@#@#@##@#@@#@@#@@@@@@@@@@@@@@@@##############@#@#@@@@@@@@@@@@@@@@@@@#@#@###########@#@#@############@#@#@#@############@@@@#@#@@@@@@@@@@@@#@#@@@@@@#@#@</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
    </property>
    <property name="buddy">
     <cstring></cstring>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="kek">
   <property name="geometry">
    <rect>
     <x>2040</x>
     <y>943</y>
     <width>2048</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="title">
    <string/>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <widget class="QLabel" name="ea6_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>2048</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <underline>false</underline>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>@###############@@@@@@@@@@@@@@@@@@@@@ ########### #@############## @#@@##@#@#@##@#@@#@@#@@@@@@@@@@@@@@@@##############@#@#@@@@@@@@@@@@@@@@@@@#@#@###########@#@#@############@#@#@#@############@@@@#@#@@@@@@@@@@@@#@#@@@@@@#@#@</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
    </property>
    <property name="buddy">
     <cstring></cstring>
    </property>
   </widget>
   <widget class="QLabel" name="ea2_4">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>10</y>
      <width>2048</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <underline>false</underline>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>@@@@########@#@#@@@@@@@@@@@@@@@@@@@@@ ########### #@############## @#@@##@#@#@##@#@@#@@#@@@@@@@@@@@@@@@@##############@#@#@@@@@@@@@@@@@@@@@@@#@#@###########@#@#@############@#@#@#@############@@@@#@#@@@@@@@@@@@@#@#@@@@@@#@#@</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
    </property>
    <property name="buddy">
     <cstring></cstring>
    </property>
   </widget>
   <widget class="QLabel" name="ea2_2">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>20</y>
      <width>2048</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <underline>false</underline>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>@@@@@@@@@@@@@@@@@@@@@ ########### #@############## @#@@##@#@#@##@#@@#@@#@@@@@@@@@@@@@@@@##############@#@#@@@@@@@@@@@@@@@@@@@#@#@###########@#@#@############@#@#@#@############@@@@#@#@@@@@@@@@@@@#@#@@@@@@#@#@</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
    </property>
    <property name="buddy">
     <cstring></cstring>
    </property>
   </widget>
   <widget class="QLabel" name="ea_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>2048</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <underline>false</underline>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>@@@@@@@@@@@@@@@@@@@@@ ########### #@############## @#@@##@#@#@##@#@@#@@#@@@@@@@@@@@@@@@@##############@#@#@@@@@@@@@@@@@@@@@@@#@#@###########@#@#@############@#@#@#@############@@@@#@#@@@@@@@@@@@@#@#@@@@@@#@#@</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
    </property>
    <property name="buddy">
     <cstring></cstring>
    </property>
   </widget>
   <widget class="QLabel" name="ea4_3">
    <property name="geometry">
     <rect>
      <x>-30</x>
      <y>40</y>
      <width>2048</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <underline>false</underline>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>@#@#@@@@@@@@@@@@@@@@@@@@@@##############@@@@@@@@@@@@@@@@@@@@@ ########### #@############## @#@@##@#@#@##@#@@#@@#@@@@@@@@@@@@@@@@##############@#@#@@@@@@@@@@@@@@@@@@@#@#@###########@#@#@############@#@#@#@############@@@@#@#@@@@@@@@@@@@#@#@@@@@@#@#@</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
    </property>
    <property name="buddy">
     <cstring></cstring>
    </property>
   </widget>
   <widget class="QLabel" name="ea5_2">
    <property name="geometry">
     <rect>
      <x>-50</x>
      <y>50</y>
      <width>2048</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <underline>false</underline>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>@#@@@@@@@@@@@@@@############@@@@@@@@@@@@##########@@@@@@@@@@@@@@@@@@@@@ ########### #@############## @#@@##@#@#@##@#@@#@@#@@@@@@@@@@@@@@@@##############@#@#@@@@@@@@@@@@@@@@@@@#@#@###########@#@#@############@#@#@#@############@@@@#@#@@@@@@@@@@@@#@#@@@@@@#@#@</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
    </property>
    <property name="buddy">
     <cstring></cstring>
    </property>
   </widget>
   <widget class="QLabel" name="ea5q_2">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>60</y>
      <width>2048</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <underline>false</underline>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>@@@@@########@@@@@@#######@@@@@#@@@@@@@@@@@@@@@@@@@@@ ########### #@############## @#@@##@#@#@##@#@@#@@#@@@@@@@@@@@@@@@@##############@#@#@@@@@@@@@@@@@@@@@@@#@#@###########@#@#@############@#@#@#@############@@@@#@#@@@@@@@@@@@@#@#@@@@@@#@#@</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
    </property>
    <property name="buddy">
     <cstring></cstring>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>860</y>
     <width>131</width>
     <height>81</height>
    </rect>
   </property>
   <widget class="QLabel" name="legs">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>  | |</string>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::LinksAccessibleByMouse</set>
    </property>
   </widget>
   <widget class="QLabel" name="body">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>221</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string> /) )\</string>
    </property>
   </widget>
   <widget class="QLabel" name="head">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>26</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>(•_•)  </string>
    </property>
   </widget>
   <widget class="QLabel" name="dia">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>0</y>
      <width>69</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Broooo((</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="ship">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>920</y>
     <width>69</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>!.!.!.!.!.!.!</string>
   </property>
  </widget>
  <widget class="QLabel" name="ship2">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>920</y>
     <width>69</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>!.!.!.!.!.!.!</string>
   </property>
  </widget>
  <widget class="QLabel" name="ship3">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>920</y>
     <width>69</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>!.!.!.!.!.!.!</string>
   </property>
  </widget>
  <widget class="QLabel" name="levl">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Player1 levl=</string>
   </property>
  </widget>
  <widget class="QLabel" name="levl2">
   <property name="geometry">
    <rect>
     <x>1100</x>
     <y>40</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Player2 levl=</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="ur">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>40</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
   <property name="intValue" stdset="0">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLCDNumber" name="ur2">
   <property name="geometry">
    <rect>
     <x>1200</x>
     <y>40</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
   <property name="intValue" stdset="0">
    <number>1</number>
   </property>
  </widget>
  <widget class="QWidget" name="widget2" native="true">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>860</y>
     <width>111</width>
     <height>81</height>
    </rect>
   </property>
   <widget class="QLabel" name="legs2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>  | |</string>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::LinksAccessibleByMouse</set>
    </property>
   </widget>
   <widget class="QLabel" name="body2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>221</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string> /) )\</string>
    </property>
   </widget>
   <widget class="QLabel" name="head2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>26</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>(•_•)  </string>
    </property>
   </widget>
   <widget class="QLabel" name="dia2">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>0</y>
      <width>69</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Broooo((</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="mob1">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>920</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0-||-|||--||---|</string>
   </property>
  </widget>
  <widget class="QLabel" name="mob2">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>920</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0-||-|||--||---|</string>
   </property>
  </widget>
  <widget class="QLabel" name="mob3">
   <property name="geometry">
    <rect>
     <x>1120</x>
     <y>920</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0-||-|||--||---|</string>
   </property>
  </widget>
  <widget class="QLabel" name="bullet3">
   <property name="geometry">
    <rect>
     <x>1100</x>
     <y>910</y>
     <width>16</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="bullet2">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>910</y>
     <width>16</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="bullet1">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>910</y>
     <width>16</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
